### ISSUE TITLE
Void function error when evaluating require form

### ISSUE THREAD

**Issue by Alexander-Miller** [rswgnu/rsw-elisp#2]
Trying to execute a `(require 'foo)` form nets me the following stacktrace:
```
Debugger entered--Lisp error: (void-function eval-expression-get-print-arguments)
  eval-expression-get-print-arguments(nil)
  rsw-elisp-internal-eval-last-sexp(nil 1)
  rsw-elisp-eval-last-sexp-or-def(nil 1)
  rsw-elisp-eval-last-sexp(nil 1)
  funcall-interactively(rsw-elisp-eval-last-sexp nil 1)
  call-interactively(rsw-elisp-eval-last-sexp record nil)
  command-execute(rsw-elisp-eval-last-sexp record)
  helm-M-x(nil #("rsw-elisp-eval-last-sexp" 0 24 (match-part "rsw-elisp-eval-last-sexp")))
  funcall-interactively(helm-M-x nil #("rsw-elisp-eval-last-sexp" 0 24 (match-part "rsw-elisp-eval-last-sexp")))
  call-interactively(helm-M-x nil nil)
  command-execute(helm-M-x)
```

Looks like `eval-expression-get-print-arguments` is not availabe in emacs 25. The fuction is fairly short so you can define it yourself for <26.

**Comment by rswgnu** [comment]
Thanks, will do and happy holidays.

-- Bob

**Comment by rswgnu** [comment]
This will be fixed in the next release.

**Comment by rswgnu** [comment]
This is fixed now in release 1.0.5.
